ncm-sysctl/src/main/perl/sysctl.pm

```diff
@@ -94,13 +91,16 @@ sub Configure {
     # execute /sbin/sysctl -p if any change made to sysctl configuration file
     #
     if ( $changes ) {
-	unless (LC::Process::trun(300,"$sysctl_exe", "-p")) {
-	    $self->error('Failed to load sysctl settings from $configFile');
-	    return;
-	}
+        $self->verbose("Changes made to $configFile, running sysctl on it");
+        my $cmd = CAF::Process->new([$sysctl_exe, '-p', $configFile],
+                                    log => $self);
+	my $output = $cmd->output;
+        $self->debug(1, "$sysctl_exe output: $output");
+        if ($? != 0) {
+            $self->error("Error loading sysctl settings from $configFile: ".
+                            "$output");
```

Comment:
`$output` is a string, and all log methods join the arguments ot a string, no need for the `.`.
```perl
$self->error("Error loading sysctl settings from $configFile: ", $output);
``` 
should be fine